lib/kotlin.gradle

```
@@ -17,6 +17,14 @@ configure(projectsWithFlags('kotlin')) {
             task.kotlinOptions.jvmTarget = target
             task.kotlinOptions.freeCompilerArgs = compilerArgs
         }
+
+        // Add the generated source directories to the source sets.
+        project.sourceSets.configureEach { sourceSet ->
+            def kotlinSrcDir = file("${project.ext.genSrcDir}/${sourceSet.name}/kotlin")
+            if (!sourceSet.java.srcDirs.contains(kotlinSrcDir)) {
```

Question) Does kotlin have its own source set? or are you adding to java intentionally?

```suggestion
            if (!sourceSet.kotlin.srcDirs.contains(kotlinSrcDir)) {
```

---

Nope. It should be `kotlin`.